### PR TITLE
add EmptyCatalog section

### PR DESCRIPTION
**Pull request by Mihail18071983**

### CODE REVIEW

File: src/app/i18n/locales/all-locales.d.ts [hunk 1]
```diff
@@ -228,6 +228,7 @@ interface AllLocalesType {
 
     delivery: string[];
     btn_to_basket: string;
+    EmptyCatalogMsgs: string[];
```
Comment (chekl) [inline src/app/i18n/locales/all-locales.d.ts]: Please, rename the variable by example empty_catalog

File: src/shared/UI/EmptyCatalog/index.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { useTranslation } from "react-i18next";
+
+export default function EmptyCatalog() {
+  const { t } = useTranslation();
+
+  const messages = t("EmptyCatalogMsgs", { returnObjects: true });
+
+  return (
+    <section className="bg-white text-secondary-900 w-full  h-full py-[118px]  font-rubik rounded-xl max-lg:ml-4 max-lg:mr-4 ">
```
Comment (chekl) [inline src/shared/UI/EmptyCatalog/index.tsx]: Please, use try to use standard numbers for padding (https://tailwindcss.com/docs/padding) for example py-28 or py-32

File: src/shared/UI/EmptyCatalog/index.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      {messages.map((message: string, index: number) => (
+        <p
+          className="text-[16px] lg:text-[24px] mb-[30px] last:mb-0 text-center odd:font-semibold even:font-light leading-7"
```
Comment (chekl) [inline src/shared/UI/EmptyCatalog/index.tsx]: Please use typography classes from styles/index.scss, and choose the most appropriate variant.
for text-[16px] use additional, for text-[24px] use secondary

**Review by chekl — CHANGES_REQUESTED** [review]

File: src/app/i18n/locales/all-locales.d.ts (hunk 1 above)
Comment (Mihail18071983) [reply src/app/i18n/locales/all-locales.d.ts]: done

File: src/shared/UI/EmptyCatalog/index.tsx (hunk 2 above)
Comment (Mihail18071983) [reply src/shared/UI/EmptyCatalog/index.tsx]: done

File: src/shared/UI/EmptyCatalog/index.tsx (hunk 3 above)
Comment (Mihail18071983) [reply src/shared/UI/EmptyCatalog/index.tsx]: done

**Pull request merged**